charts/ratify/templates/_helpers.tpl

```diff
@@ -113,7 +113,7 @@ tls.crt: {{ $top.Values.provider.tls.crt | b64enc | quote }}
 tls.key: {{ $top.Values.provider.tls.key | b64enc | quote }}
 ca.crt: {{ $top.Values.provider.tls.caCert | b64enc | quote }}
 ca.key: {{ $top.Values.provider.tls.caKey | b64enc | quote }}
-{{- else }}
+{{- else if not $top.Values.featureFlags.RATIFY_CERT_ROTATION }}
```

Comment:
how about for upgrade scenarios? if I had a cert rotator available before and then I upgrade, it will recreate the ratify-tls secret but this time with an empty secret yet again. This is problematic for in flight requests being processed by existing ratify instance while the new one boots up. We should consider changing the helm hook for the ratify-tls secret to be pre-install only.